### DISCUSSION TITLE
Create a bill from a transaction

### DISCUSSION THREAD

**Discussion by mhriemers** [firefly-iii/firefly-iii#9366] [Firefly III]
Is it possible to create a bill from a transaction, easily? I find myself having two windows open constantly and copy-pasting a lot of information.

**Comment by JC5** [comment]
There is a ticket for this, but it's not yet possible I'm afraid.

**Comment by floriskruisselbrink** [comment]
I'm following this.

The biggest problem I have with the two-window workaround (one for viewing transactions, one for creating the bills) is that firefly-iii itself gets its navigation completely screwed up. For example:
- open firefly-iii (lets call this window 1)
- you open a second window with firefly-iii and navigate to the bills section (window 2)
- in window 1 you navigate to the transaction you want to use as base for a new bill
- in window 2 you create a new bill and save it. Now you end up in the transaction-screen instead of the bills section where you started.

I think the logic firefly-iii uses to determine what page to show after closing an edit-window (works the same for transactions, bills, budgets, whatever) is to check wat page was last viewed before, but the catch is it checks that globally in your session, and not in the history of the current window.

Probably the above situation is a completely different issue / bug / 'feature', but I find it a little annoying at times.
